### DISCUSSION TITLE
Confused about having multiple climate entities in the climate page

### DISCUSSION THREAD

**Discussion by thromera** [Blackymas/NSPanel_HA_Blueprint#2254] [General]
Hello! I'm quite a beginner using the NSPAnel, pardon me if the information is present somewhere. 

I'm trying to control many climate entities from the climate page (when you click on the home temperature on the homepage). 

I've only been successful in adding the `main climate entity` in the configuration, however I would love to be able to control all my 4 AC units from this page (and not use a dedicated "button" page to do so). 

![CleanShot 2024-08-30 at 18 52 35@2x](https://github.com/user-attachments/assets/024cfa14-430f-4760-bcde-3d63179d3fe3)

![CleanShot 2024-08-30 at 18 57 12@2x](https://github.com/user-attachments/assets/ed7ba43a-1ad5-456d-aa68-1c7cdd2fac99)

Am I missing something? It's maybe just not possible right now, and I must add all my clim entities to a `button page` instead? 

Thanks! 

Side note: The [Github documentation is not up-to-date](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/blueprint.md#climate-to-control-optional). It mentions the following: 
> Additional climate entities can be assigned to buttons and will be explained later in this document.

But there are no explanations later on 😢. (or maybe you meant "hardware buttons"?)

**Comment by edwardtfn** [comment]
Unfortunately we cannot call a detailed page (like que climate page) from another detailed page, as we loose the reference for the previous page. Probably not that difficult to solve, but would require some investigations on all the different cases.

For now, you can assign climate entities to any of the buttons (custom buttons, hardware buttons or button pages) and then have multiple climate entities controlled by the same panel, but not a link inside another climate page.

**Reply by thromera** [reply]
Ok, thanks. 

If I rephrase to make sure I understood properly, from the climate page (the page you access when clicking on the home temperature on the homepage), it's not possible to add a button to another entity, because it would open another page to that entity, and that is not supported as of today. 

The workaround is to use the regular pages (the ones you access when swiping left/right/up/down from the homepage)?

Is that correct? 

That would be a nice feature, to be able to control all climate entities, from the climate page ☺️. 

Thanks for the explanation, I feel like this could be explained in the documentation as a gotcha, I will open a PR to update it.

**Reply by thromera** [reply]
Clarifications in the documentation: https://github.com/Blackymas/NSPanel_HA_Blueprint/pull/2256
I've also created an enhancement issue: https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/2257

**Reply by edwardtfn** [reply]
Yeah! That's correct.
I will take a look on how to implement this later...

**Reply by Tontze** [reply]
I would like to control three AC´s .. Is it possible just add second and third "main" climate pages ? Or option to duplicate current one  ?

**Reply by gchaviaras1** [reply]
The main climate entity is a single one entity inside your nspanel configuration which reflects a climate entity from your HA. 
A workaround is the following: you can control multiple climate entities grouping them in home assistant under one single climate entity ( awesome thermostat, better thermostat etc ) and add it in the main climate entity.
Else you can add the thermostats from your ACs as button entities in the nspanel configuration
